labs/db-iterator/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@reshuffle/labs-db-iterator",
+  "version": "0.0.1",
+  "description": "Iterator for large DB queries",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "lint": "eslint -c ../../common/.eslintrc.json --ext .ts src",
+    "test": "RESHUFFLE_DB_BASE_URL=a RESHUFFLE_APPLICATION_ID=b RESHUFFLE_APPLICATION_ENV=c RESHUFFLE_ACCESS_TOKEN=d ava"
```

Comment:
Yuck, thanks!  Done.
Unfortunately we do need this because we always load the client.  I think I'll switch the default DB client to do "lazy" loading of environment variables via an access function; that way if you don't use the default client you won't need these variables.  That will go in a later PR.